src/api/java/baritone/api/cache/IWaypoint.java

```diff
@@ -66,6 +66,16 @@
          */
         HOME("home", "base"),
 
+        /**
+         * Tag indicating a position explictly marked as a chest
+         */
+        CHEST("chest"),
+
+        /**
+         * Tag indicating a position explictly marked to use a chest
+         */
+        USECHEST("usechest"),
```

Comment:
Why do you have to set the chest position and the usechest location? Can't we use GoalGetToBlock on the chest??

Reply:
I thougth if i use GoalGetToBlock baritone would destroy the chest block,but your suggestion works fine